Examples/Physics_applications/spacecraft_charging/PICMI_inputs_rz.py

```diff
@@ -46,11 +47,11 @@ def correct_space_charge_fields(self, q=None):
             q = compute_actual_charge_on_spacecraft()
 
         # Correct fields so as to recover the actual charge
-        Er = ExWrapper(include_ghosts=True)[:,:]
-        Er[...] = Er[...]+(q - q_v)*self.normalized_Er[...]
-        Ez = EzWrapper(include_ghosts=True)[:,:]
+        Er = ExWrapper(include_ghosts=True)
+        Er[...] += (q - q_v)*self.normalized_Er[...]
```

Comment:
This could be cleaned up a bit more. Since `self.normalized_Er` is an already a numpy array, the `[...]` is not needed. And the same for `Ez` and `phi` below.